app/javascript/components/MapView.js

```diff
@@ -30,14 +30,15 @@ class MapView extends React.Component {
         var arrayLength = props.locations.length;
         for (var i = 0; i < arrayLength; i++) {
           if (props.locations[i].floor_id == props.current_selected_floor) {
+            alert(JSON.stringify(props.locations[i]))
```

Comment:
don't leave debugging alerts.. or switch it to console.log for debugging at least